src/Particles/babylon.particleSystem.ts

```diff
@@ -446,20 +467,6 @@
             this._indexBuffer = this._scene.getEngine().createIndexBuffer(indices);
         }
 
-        /**
-         * "Recycles" one of the particle by copying it back to the "stock" of particles and removing it from the active list.
-         * Its lifetime will start back at 0.
-         * @param particle The particle to recycle
-         */
-        public recycleParticle(particle: Particle): void {

```

Comment:
This has to remain the same

Reply:
But I created a function variable so the consumer won't feel the difference :) if I have to keep it a function I will do a if else inside the recycleparticle and I'm trying to avoid that as much as possible :)

Reply:
But you should not use root PS to create particles for sub emitters. They have to be independant